apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -45,6 +46,15 @@ const CurriculumCatalog = ({
     setAssignSuccessMessage('');
   };
 
+  const getQuickViewState = () => {
+    const urlParams = queryParams();
+    return 'quick_view' in urlParams && urlParams['quick_view'] === 'true'
+      ? true
+      : false;
+  };
+
+  const isQuickViewDisplayed = getQuickViewState();
```

Comment:
I think this can all be simplified to
`const getQuickViewState = () => {
const urlParams = queryParams();
return urlParams['quick_view'] === 'true'
}`